### PR TITLE
Workbook v1.1 / Improved Queries

### PR DESCRIPTION
**Pull request by arthurclares**
Added checks for Compute, Storage and Networking.
Also added "quick fix" for Idle Disks.

## 🛠️ Description
Added new features to workbook for Compute, Networking and Storage.
Improved filter by tag. 
Added "quick fix" to Idle disks.

Fixes #(issue)
NA

## 🔬 How has this been tested?

- [ ] 🫰 PS -WhatIf / az validate
- [ x] 👍 Manually deployed + verified
- [ ] 💪 Unit tests

### CODE REVIEW

File: src/workbooks/optimization/workbook.json [hunk 1]
```diff
@@ -2,268 +2,16 @@
...
...Ps|\r\n|v1.1.8| 16/02/2022 | -Added untagged resources list <br> -Added storage account tiers <br> -Added a link to storage account insights for each legacy storage account <br> -Added AHUB check for windows VMSS <br> -Fixed unattached disk query <br> -Fixed App service query (not showing all apps in a service plan) <br> -Added \"Cost Information\" tab|\r\n|v1.1.9| 14/06/2022 | - Fixed \"unattached disk\" query in the cost information tab |\r\n|v1.2.0| 24/07/2022 | - Removed FileStorage account type from V2 query <br> - Added new column in \"Cost Information\" tab to show total VM cost with AHUB <br> - Changed VPN metrics aggregation <br> - Changed column some names |\r\n|v1.2.1| 08/11/2022 | - Add Front Door metrics <br> - Changed cost information billing for VMs without AHUB to TheLastBillingMonth <br> - Increased row limit to in compute queries  <br> - Updated AKS query to show Spot VMs  <br> - Fixed Linux AHUB query to show RHEL license <br> - Add subscriptions advisor recommendations |\r\n|v1.2.2| December/2022 | - Minor fix to the Storage Advisor query |\r\n|v1.3.0 | March /2023 | - Fixed Cost Information Tab |\r\n|v2.0.0 | March /2023 | - Added Azure Hybrid Benefit tab|\r\n"
```
Comment (flanakin) [inline src/workbooks/optimization/workbook.json:7]: A few thoughts:
1. "Assessment" is a bit of a loaded term in FinOps. Should we use a different term?
2. We should put these details into the toolkit changelog on Github.
3. Should we include the changelog here or just link to the toolkit changelog? Linking out might be a way to drive discoverability of new versions.
4. Side note: You might be able to compare the current workbook version with what's released on Github to introduce an update notification. Something to think about for the future.

**Comment by flanakin** [comment]
This is a really big change. We should come up with a plan for how to make workbook PRs easier to review. Maybe that just means smaller PRs for now. So much changes there's pretty much no traceability back to older states. We should also talk to the workbook team to see if they have a solution to that.

File: src/workbooks/optimization/workbook.json (hunk 1 above)
Comment (arthurclares) [reply src/workbooks/optimization/workbook.json]: I have changed the workbook name to Azure Cost Optimization Workbook

File: src/workbooks/optimization/workbook.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
```
Comment (flanakin) [inline src/workbooks/optimization/workbook.json:7]: You changed it back to assessment 😅

**Review by flanakin — CHANGES_REQUESTED** [review]

File: src/workbooks/optimization/workbook.json (hunk 2 above)
Comment (arthurclares) [reply src/workbooks/optimization/workbook.json]: Sorry, my template was outdated.
Just fixed it.

**Review by flanakin — APPROVED** [review]

**Pull request merged**